sinks/cortex/cortex.go

```diff
@@ -156,10 +156,14 @@ func (s *CortexMetricSink) Flush(ctx context.Context, metrics []samplers.InterMe
 	req.Header.Set("User-Agent", "veneur/cortex")
 	req.Header.Set("X-Prometheus-Remote-Write-Version", "0.1.0")
 
-	r, err := s.client.Do(req)
+	ts := time.Now()
+	r, err := s.Client.Do(req)
 	if err != nil {
 		span.Error(err)
-		s.logger.WithError(err).Info("Failed to post request")
+		s.logger.WithFields(logrus.Fields{
```

Comment:
This should be a log severity higher than "Info", probably "Error".

I'd also be in support to adding a `.Seconds()` to make the logs more readable.